keras_cv/models/mobilenet_v3.py

```diff
@@ -0,0 +1,557 @@
+    )(x)
+    x = activation(x)
+    if include_top:
+        x = layers.GlobalAveragePooling2D(keepdims=True)(x)
+        x = layers.Conv2D(
+            last_point_ch,
+            kernel_size=1,
+            padding="same",
+            use_bias=True,
+            name="Conv_2",
+        )(x)
+        x = activation(x)
+
+        if dropout_rate > 0:
+            x = layers.Dropout(dropout_rate)(x)
+        x = layers.Conv2D(num_classes, kernel_size=1, padding="same", name="Logits")(x)
+        x = layers.Flatten()(x)
+        x = layers.Activation(activation=classifier_activation, name="Predictions")(x)
+    elif pooling == "avg":
+        x = layers.GlobalAveragePooling2D(name="avg_pool")(x)
+    elif pooling == "max":
+        x = layers.GlobalMaxPooling2D(name="max_pool")(x)
+
+    model = keras.Model(inputs, x, name=name, **kwargs)
+
+    if weights is not None:
+        model.load_weights(weights)
+    return model
+
+
+def MobileNetV3Small(
+    input_shape=(None, None, 3),
+    alpha=1.0,
+    minimalistic=False,
+    include_top=True,
+    weights=None,
+    num_classes=None,
+    pooling=None,
+    dropout_rate=0.2,
+    classifier_activation="softmax",
+    include_rescaling=True,
+    name="MobileNetV3Small",
+    **kwargs,
+):
+    def stack_fn(x, kernel, activation, se_ratio):
+
+        x = InvertedResBlock(1, depth(16 * alpha), 3, 2, se_ratio, layers.ReLU(), 0)(x)
+        x = InvertedResBlock(
+            72.0 / 16, depth(24 * alpha), 3, 2, None, layers.ReLU(), 1
+        )(x)
+        x = InvertedResBlock(
+            88.0 / 24, depth(24 * alpha), 3, 1, None, layers.ReLU(), 2
+        )(x)
+        x = InvertedResBlock(4, depth(40 * alpha), kernel, 2, se_ratio, activation, 3)(
+            x
+        )
+        x = InvertedResBlock(6, depth(40 * alpha), kernel, 1, se_ratio, activation, 4)(
+            x
+        )
+        x = InvertedResBlock(6, depth(40 * alpha), kernel, 1, se_ratio, activation, 5)(
+            x
+        )
+        x = InvertedResBlock(3, depth(48 * alpha), kernel, 1, se_ratio, activation, 6)(
+            x
+        )
+        x = InvertedResBlock(3, depth(48 * alpha), kernel, 1, se_ratio, activation, 7)(
+            x
+        )
+        x = InvertedResBlock(6, depth(96 * alpha), kernel, 2, se_ratio, activation, 8)(
+            x
+        )
+        x = InvertedResBlock(6, depth(96 * alpha), kernel, 1, se_ratio, activation, 9)(
+            x
+        )
+        x = InvertedResBlock(6, depth(96 * alpha), kernel, 1, se_ratio, activation, 10)(
+            x
+        )
+        return x
+
+    return MobileNetV3(
+        stack_fn,
+        1024,
+        input_shape,
+        alpha,
+        include_top,
+        weights,
+        num_classes,
+        pooling,
+        dropout_rate,
+        classifier_activation,
+        include_rescaling,
+        minimalistic,
+        name=name,
+        **kwargs,
+    )
+
+
+def MobileNetV3Large(
+    input_shape=(None, None, 3),
```

Comment:
please update all constructors to match the API of DenseNet.

https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/densenet.py#L255

include_rescaling, include_top required.  IF include_top true, num_classes required.

Also update the underlying helper to match this API

Reply:
Gotcha